scripts/quickstart.py

```diff
@@ -0,0 +1,427 @@
+#!/usr/bin/env python3
+"""
+Quickstart script: Full workflow for web-hacker
+This script guides you through: Launch Chrome → Monitor → Discover → Execute
+"""
+
+import os
+import sys
+import time
+import platform
+import subprocess
+import shutil
+from pathlib import Path
+from typing import Optional
+from urllib.parse import quote
+
+try:
+    import requests
+except ImportError:
+    print("Error: 'requests' package is required. Install it with: pip install requests")
+    sys.exit(1)
```

Comment:
We already list `requests` in `pyproject.toml`.

```suggestion
import requests
```